Source/RxBlocking/RxNimble.swift

```diff
@@ -1,95 +0,0 @@
-import RxSwift
-import RxBlocking
-import Nimble
-
-// This is handy so we can write expect(o) == 1 instead of expect(o.value) == 1 or whatever.
-@available(*, deprecated, message: "use expect(o).first instead")
-public func equalFirst<T: Equatable, O: ObservableType>(_ expectedValue: T?) -> Predicate<O> where O.E == T {
```

Comment:
why not leave the deprecations in place? I don't entirely mind but usually removing code isn't a proper deprecation path.

Reply:
They were deprecated in a previous major and the author is suggesting to get rid of them as part of the major update, makes sense to me

Reply:
@freak4pc yes, i agree with @gobetti. They were deprecated two years ago. I think it's time to get rid of them